components/input/src/input/input.js

```diff
@@ -187,6 +200,14 @@ export class Input extends Component {
                         'read-only': readOnly,
                     })}
                 />
+                {clearable && value.length ? (
+                    <button onClick={clearText}>
```

Comment:
I think we should add a `className`, and `type=button`. If this doesn't have a type, it will default to `submit`, and post if it's within a form.